modules/microshift-configuration.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+//* microshift_support/microshift-etcd.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-configuration_{context}"]
+= Configure `memoryLimitMB` to set parameters for the {product-title} etcd server
+
+By default, etcd will use as much memory as necessary to handle the load on the system. In some memory constrained systems it might be necessary to limit the amount of memory etcd is allowed to use at a given time.
+
+.Procedure
+
+* Edit `/etc/microshift/config.yaml` to set the `memoryLimitMB` value.
++
+[source,yaml]
+----
+etcd:
+  memoryLimitMB: 128
+----
++
+[NOTE]
+====
+128mb is the minnimum permissable value for memoryLimitMB on {product-title}. Values close to the minnimum value is more likely to impact etcd performance. The lower the limit, the longer it will take to respond to queries or time requests out if the limit is low and the etcd usage is high.
+====
+
+.Verification
+
+. After modifying the `memoryLimitMB` value in `/etc/microshift/config.yaml`, restart {product-title} by running the following command:
+----
+$ sudo systemctl restart microshift
+----
+
+. Verify the new `memoryLimitMB` value is in use by running the following command:
```

Comment:
Do you need root privileges for this? I'm guessing no but wanted to double check. If you do you could add a prerequisites to the procedure.